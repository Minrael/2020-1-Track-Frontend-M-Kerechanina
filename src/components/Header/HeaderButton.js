import React from 'react'

export default function HeaderButton ({name, src}) {
    return(
    <div className = { name }>
      <img alt = { name } src = { src }/>
    </div>)
}
