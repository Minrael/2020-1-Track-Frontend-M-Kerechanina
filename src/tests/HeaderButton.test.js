import React from 'react'
import renderer from 'react-test-renderer'
import HeaderButton from '../components/Header/Header'
import search from '../static/search.svg';

it('renders correctly', () => {
    const SearchButton = renderer
    .create(<HeaderButton name = 'search-button' src = {search}/>)
    .toJSON();
    expect(SearchButton).toMatchSnapshot();
})