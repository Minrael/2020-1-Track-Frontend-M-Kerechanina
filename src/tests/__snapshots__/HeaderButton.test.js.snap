// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <div
    className="back-button"
  >
    <img
      alt="back-button"
      src="left-arrow-key.svg"
    />
  </div>
  <div
    className="user-info"
  >
    <div
      className="divAvatar"
    >
      <img
        alt="avatar"
        className="avatar"
        src="american.svg"
      />
    </div>
    <div
      className="name"
      styles={Object {}}
    >
      <div
        className="user-name"
      >
        Дженнифер
      </div>
      <div
        className="last-visit"
      >
        была 2 часа назад
      </div>
    </div>
  </div>
  <div
    className="search-button"
  >
    <img
      alt="search-button"
      src="search.svg"
    />
  </div>
  <div
    className="more-button"
  >
    <img
      alt="more-button"
      src="more.svg"
    />
  </div>
</div>
`;
